src-tauri/capabilities/desktop.json

```diff
@@ -0,0 +1,11 @@
+{
+  "identifier": "desktop-capability",
```

Comment:
why we need this one? while we don't expose any command called desktop-capability.